README.md

```diff
@@ -41,6 +41,20 @@ tmpl.render({"what", "World"}, [&ss](const std::string& str) {
 // ss.str() == "Hello World!"
 ````
 
+### Usage with cmake
+
+```cmake
+find_package(mustache REQUIRED)
```

Comment:
possible to use namespace here somehow?

Reply:
we could change this to something like: `find_package(nlohmann_json 3.2.0 REQUIRED)`

ref: 
https://github.com/nlohmann/json

Reply:
yeah that might be better.